frontend/src/bundles/recently-viewed/store/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { type AsyncThunkConfig } from '~/bundles/common/types/types.js';
+
+import { type RecentlyViewedTemplatesResponseDto } from '../types/types';
+import { name as sliceName } from './slice.js';
+
+const getRecentlyViewedTemplates = createAsyncThunk<
+    RecentlyViewedTemplatesResponseDto[],
+    undefined,
+    AsyncThunkConfig
+>(`${sliceName}/templates`, async (_, { extra }) => {
+    const { recentlyViewedApi } = extra;
+
+    return await recentlyViewedApi.getRecentlyViewedTemplates();
```

Comment:
```suggestion
    return recentlyViewedApi.getRecentlyViewedTemplates();
```
return await is redundant